how2meet/ui/components/elements.py

```diff
@@ -12,29 +12,29 @@
     "burnt-sienna": "#E76F51"
     }
 
-def button(*args, **kwargs):
+def button(*args, **kwargs) -> ui.button:
     return ui.button(*args, **kwargs).props("outline color=white")
 
-def checkbox(*args, **kwargs):
+def checkbox(*args, **kwargs) -> ui.checkbox:
     return ui.checkbox(*args, **kwargs).props("flat color=white")
 
-def input(*args, **kwargs):
+def input(*args, **kwargs) -> ui.input:
     return ui.input(*args, **kwargs).props("filled color=white")
 
-def label(*args, **kwargs):
+def label(*args, **kwargs) -> ui.label:
     return ui.label(*args, **kwargs).props("color=white")
 
-def card(*args, **kwargs):
+def card(*args, **kwargs) -> ui.card:
     return ui.card(*args, **kwargs).props("flat color=dark")
 
-def dialog(*args, **kwargs):
+def dialog(*args, **kwargs) -> ui.dialog:
     return ui.dialog(*args, **kwargs).props("color=dark")
 
-def textarea(*args, **kwargs):
+def textarea(*args, **kwargs) -> ui.textarea:
     return ui.textarea(*args, **kwargs).props("filled color=white")
 
-def date(*args, **kwargs):
-    return ui.date(*args, **kwargs).props("minimal")
+def date(*args, **kwargs) -> ui.date:
+    return ui.date(*args, **kwargs).props("minimal navigation-min-year-month=1970/01")
```

Comment:
Would it make sense to make the minimum date be the current day? Doesn't make sense to plan an event in the past IMO

Reply:
What are you the time cops?